src/Symfony/Component/HttpKernel/Tests/KernelTest.php

```diff
@@ -626,7 +626,7 @@ protected function getBundle($dir = null, $parent = null, $className = null, $bu
     {
         $bundle = $this
             ->getMockBuilder('Symfony\Component\HttpKernel\Bundle\BundleInterface')
-            ->setMethods(['getPath', 'getPublicDir', 'getParent', 'getName'])
+            ->setMethods(['getPath', 'getParent', 'getName'])
```

Comment:
isn't `getParent` removed as well ?

Reply:
It's only deprecated and will be removed in 5.0, right?

Reply:
`getPublicDir()` was planned to be introduced with #31975 in 4.4 and was removed again in #32845

Reply:
deprecated in 3.4 and removed in 4.0